### DISCUSSION TITLE
Strange behave of NumberInput

### DISCUSSION THREAD

**Discussion by skelawsky** [mantinedev/mantine#1628] [Q&A]
Why is NumberInput behaving differently? Is this desired behavior? 
Value goes over the limit with backend validation.

[Video showing diffrence](https://www.youtube.com/watch?v=jAvP0k4wLiY)

```
<input
type="number"
value={element?.quantity}
onChange={(e) =>
updateZKItem({ zkitem_id: element.id, data: { quantity: e.target.value } })
}
/>


<NumberInput
value={element?.quantity}
noClampOnBlur
onChange={(val) =>
updateZKItem({ zkitem_id: element.id, data: { quantity: val } })
}
/>
```

**Comment by rtivital** [comment]
Have you set min/max props?

**Reply by skelawsky** [reply]
@rtivital 
Unfortunately, I do not have access to max value. Regular input works without this prop.
